packages/react/src/components/RadioTile/RadioTile.tsx

```diff
@@ -21,12 +21,73 @@ import { noopFn } from '../../internal/noopFn';
 import { Text } from '../Text';
 import { useFeatureFlag } from '../FeatureFlags';
 
+export interface RadioTileProps {
+  /**
+   * Specify whether the `RadioTile` should be checked.
+   */
+  checked?: boolean;
+
+  /**
+   * The `RadioTile` content.
+   */
+  children?: React.ReactNode;
+
+  /**
+   * Provide an optional `className` to be applied to the underlying `<label>`.
+   */
+  className?: string;
+
+  /**
+   * Specify whether the `RadioTile` should be disabled.
+   */
+  disabled?: boolean;
+
+  /**
+   * Provide a unique id for the underlying `<input>`.
+   */
+  id?: string;
+
+  /**
+   * `true` to use the light version. For use on $ui-01 backgrounds only.
```

Comment:
I believe [`ui-01` has become `layer-01`](https://github.com/carbon-design-system/carbon/blob/main/docs/migration/v11.md#design-tokens) in Carbon 11.

```suggestion
   * `true` to use the light version. For use on $layer-01 backgrounds only.
```